crypto/tss/signer/signer.go

```diff
@@ -26,22 +27,26 @@ import (
 	"github.com/getamis/sirius/log"
 )
 
+var (
+	ErrInconsistentPeerNum = errors.New("inconsistent peer num")
```

Comment:
This error has been defined in `tss/utils.go`

Reply:
OK